src/Pages/Shop/ShopListRow.scss

```
@@ -0,0 +1,58 @@
+@font-face {
+  font-family: "Neue Haas Grotesk-Bold";
+  src: local("Neue-HaasGrotesk-Text"), url("/public/") format("woff2"),
+    url("http://localhost:3000/Font/NeueHaasGroteskText-Bold.otf")
```

```suggestion
    url("/Font/NeueHaasGroteskText-Bold.otf")
```
경로 이렇게 설정해두면 aws 올려서 배포하면 다 깨집니다!

---

왜그런가요..? 궁금..수정했습니다